src/openapi.yaml

```diff
@@ -115,17 +115,77 @@ paths:
       tags:
         - catalogussen
     parameters:
+      - name: id
+        in: path
+        description: A unique integer value identifying this Zaaktype.
```

Comment:
ID documentatie is zeker automatisch? Andere descriptions zijn Nederlands.

Reply:
Yep, ga ik binnenkort (TM) eens een mep op geven, komt goed :)